schemas/filing-received.go

```diff
@@ -0,0 +1,35 @@
+package schemas
+
+//  FilingReceived struct for filing-received topic
+type FilingReceived struct {
+	ApplicationID string                   `avro:"application_id"                  json:"application_id"`
+	Attempt       int32                    `avro:"attempt"                         json:"attempt"`
+	ChannelID     string                   `avro:"channel_id"                      json:"channel_id"`
+	Presenter     PresenterFilingRecieved  `avro:"presenter"                       json:"presenter"`
+	Submission    SubmissionFilingRecieved `avro:"published_at"                    json:"published_at"`
+	Items         []Items                  `avro:"items"                           json:"items"`
+}
+
+//  PresenterFilingReceived struct represents PresenterFilingReceived in the FilingReceived struct
+type PresenterFilingRecieved struct {
+	Forename string `avro:"forename"                        json:"forename"`
+	Language string `avro:"language"                        json:"language"`
+	Surname  string `avro:"surname"                         json:"surname"`
+	UserID   string `avro:"user_id"                         json:"user_id"`
+}
+
+//  SubmissionFilingReceived struct represents SubmissionFilingReceived in the FilingReceived struct
```

Comment:
Please use only a single space after the double slash.

Reply:
Remove in [ce005fb](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/ce005fb8926cbc2899d61cf66de233a4e80ce97c)

Reply:
Not done.

Reply:
double spaces removed in [9edd492](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/9edd49223eaff7f92ad875ac4925a15528511c59)